src/arch/xtensa/schedule/task.c

```diff
@@ -16,7 +16,6 @@
 #include <sof/lib/cache.h>
 #include <sof/lib/cpu.h>
 #include <sof/lib/memory.h>
-#include <sof/lib/wait.h>
```

Comment:
commit typo: r_yield() -> k_yield()

Reply:
Oops. I wonder why this did not fail in CI build. Hope there are no more such errors.